src/app/give-help/material-aid-form/material-aid-form.component.ts

```diff
@@ -25,7 +25,7 @@ export class MaterialAidFormComponent {
   phonePrefix: string = '48';
   phoneNumber: string = '';
   loading: boolean = false;
-  @ViewChild('phoneInput') phoneInput!: ElementRef;
+  @ViewChild('phoneInput') phoneInput!: ElementRef<HTMLInputElement>;
```

Comment:
Ta zmiana sprawia, że `this.phoneInput.nativeElement` nie jest już `any`, ale `HTMLInputElement`.